retry.go

```diff
@@ -44,6 +44,12 @@ SEE ALSO
 * [matryer/try](https://github.com/matryer/try) - very popular package, nonintuitive interface (for me)
 
 BREAKING CHANGES
```

Comment:
Personally, I think that these comments do not belong to the code, it is already in the Readme.md file.  ... and it is usually placed into CHANGELOG.md.

Reply:
Readme is generated from code.